src/tests/dbus-tests/run_tests.py

```diff
@@ -103,6 +103,8 @@ def install_config_files(projdir, tmpdir):
     copied.extend(_copy_files(policies, '/usr/share/polkit-1/actions/', tmpdir))
 
     # udisks2.conf
+    if not os.path.exists('/etc/udisks2'):
+        os.mkdir('/etc/udisks2', 0o755)
```

Comment:
Shouldn't the directory be removed afterwards if it was created by the tests?

Reply:
Yes, but the existing "cleaning" code works for files only and it's already too complicated and I didn't want to make it more complicated. And I don't think it's a big issue to create this directory without cleaning it (it's just an empty directory, not a config file that can change something).

Reply:
Sounds reasonable to me.